### PR TITLE
chore(TDOPS-2955): update default values of locales

### PR DESCRIPTION
**Pull request by ybaskaran**
**What is the problem this PR is trying to solve?**
Default values in translation expressions are not up-to-date
https://jira.talendforge.org/browse/TDOPS-3002

**What is the chosen solution to this problem?**
update default values of locales using jscodeshift

**Please check if the PR fulfills these requirements**

- [ ] The PR have used `yarn changeset` to a request a release from the CI if wanted.
- [ ] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features) And [non reg](./screenshots.md) done before need review
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

**[ ] This PR introduces a breaking change**

### CODE REVIEW

**Review by jmfrancois — APPROVED** [review]

File: packages/components/src/List/ListComposition/ItemsNumber/ItemsNumber.component.js [hunk 1]
```diff
@@ -10,13 +10,13 @@ function ItemsNumber(props) {
...
 				  t('LIST_TOOLBAR_NUMBER_OF_SELECTED_ITEMS', {
-						defaultValue: '{{count}}/{{total}} items',
+						defaultValue: '{{count}}/{{total}} item',
 						count: selected,
 						total: totalItems,
 				  })
 				: label ||
 				  t('LIST_TOOLBAR_TOTAL_NUMBER_OF_ITEMS', {
-						defaultValue: '{{count}} items',
+						defaultValue: '{{count}} item',
```
Comment (lmaillet) [inline packages/components/src/List/ListComposition/ItemsNumber/ItemsNumber.component.js:19]: No more `s` 🤔

File: packages/components/src/ObjectViewer/JSONLike/JSONLike.component.js [hunk 2]
```diff
@@ -361,7 +361,7 @@ function UntranslatedComplexItem(props) {
 						key="badge"
 						className={`${theme.badge} badge`}
 						aria-label={t('TC_OBJECT_VIEWER_NB_CHILD', {
-							defaultValue: 'Contains {{count}} child',
+							defaultValue: 'Contains {{count}} child object',
```
Comment (lmaillet) [inline packages/components/src/ObjectViewer/JSONLike/JSONLike.component.js:364]: `objects` ?

File: packages/components/src/ResourceList/ResourceList.stories.js [hunk 3]
```diff
@@ -259,7 +259,7 @@ export function FilteredResourceList(props) {
 			toolbar={{
 				name: {
 					label: t('FILTER', {
-						defaultValue: 'Find a {{type, lowercase}}',
+						defaultValue: 'Filter:',
 						type: props?.renderAs?.name,
```
Comment (lmaillet) [inline packages/components/src/ResourceList/ResourceList.stories.js:263]: Why a `type` parameter if the defaultValue is 'Filter' ?

**Review by lmaillet — COMMENTED** [review]
Maybe you can ask for someone from the translation team for a review?

**Comment by ybaskaran** [comment]
> Maybe you can ask for someone from the translation team for a review?

yes, some translations are not right, i will check with the Translation team

File: packages/components/src/List/ListComposition/ItemsNumber/ItemsNumber.component.js (hunk 1 above)
Comment (sbouyer) [reply packages/components/src/List/ListComposition/ItemsNumber/ItemsNumber.component.js:19]: correct! The plural is added in LIST_TOOLBAR_TOTAL_NUMBER_OF_ITEMS_plural.

File: packages/components/src/ObjectViewer/JSONLike/JSONLike.component.js (hunk 2 above)
Comment (sbouyer) [reply packages/components/src/ObjectViewer/JSONLike/JSONLike.component.js:364]: Same as above, there is a separate entry for plural : TC_OBJECT_VIEWER_NB_CHILD_plural

File: packages/components/src/List/ListComposition/ItemsNumber/ItemsNumber.component.js (hunk 1 above)
Comment (ybaskaran) [reply packages/components/src/List/ListComposition/ItemsNumber/ItemsNumber.component.js:20]: added defaultValue_plural

File: packages/components/src/ObjectViewer/JSONLike/JSONLike.component.js (hunk 2 above)
Comment (ybaskaran) [reply packages/components/src/ObjectViewer/JSONLike/JSONLike.component.js:364]: defaultValue_plural

File: packages/components/src/ResourceList/ResourceList.stories.js (hunk 3 above)
Comment (ybaskaran) [reply packages/components/src/ResourceList/ResourceList.stories.js]: removed type param

**Review by lmaillet — APPROVED** [review]

**Pull request merged**